addons/im_livechat/static/src/js/im_livechat.js

```diff
@@ -353,8 +353,14 @@ var LivechatButton = Widget.extend({
      */
     _onNotification: function (notifications) {
         var self = this;
+        var message_ids = [];
```

Comment:
i think best fix is to keep handling the notification, and in the `else` case of the `_handleNotification()` have something like this:

```js
if (this._messages.some(message => message.getID() === notification[1].id)) {
   return;
}
```

Basically, if we receive a notification related to message that already exists, stop handling the notification.

Reply:
Thanks ! :) Yeah better in handle notification method indeed !